fdbserver/RestoreApplier.actor.cpp

```diff
@@ -101,7 +101,7 @@ ACTOR Future<Void> restoreApplierCore(RestoreApplierInterface applierInterf, int
 
 // The actor may be invovked multiple times and executed async.
 // No race condition as long as we do not wait or yield when operate the shared data.
-// Multiple such actors can run on different fileIDs, because mutations in different files belong to different versions;
+// Multiple such actors can run on different fileIDs;
```

Comment:
Add a comment line:
Mutations in different files may belong to same or different versions.

Reply:
Done.